mpqp/core/instruction/gates/native_gates.py

```diff
@@ -150,7 +150,10 @@ def to_other_language(
     ):
         if qiskit_parameters is None:
             qiskit_parameters = set()
-        theta = float(self.theta) if self._numeric_parameters else self.theta
+        try:
```

Comment:
since you changed this did you check if `_numeric_parameters` is still needed ?

Reply:
I checked, it's not used anywhere, I removed it